File: android/src/main/kotlin/com/seewo/flutter/video_native_preview/IMediaPlayerOption.java
package com.seewo.flutter.video_native_preview;

public interface IMediaPlayerOption {
    void doPauseResume();
    void seekTo(boolean fromUser, int msec);
    void updateProgress(boolean fromUser, int msec);
    void onScreenOrientation(boolean landscape);
    void onTopBarVisibility(boolean visible);
}

File: android/src/main/kotlin/com/seewo/flutter/video_native_preview/IMediaPlayerOption.kt
package com.seewo.flutter.video_native_preview

interface IMediaPlayerOption {
    fun doPauseResume()
    fun seekTo(fromUser: Boolean, msec: Int)
    fun updateProgress(fromUser: Boolean, msec: Int)
    fun onScreenOrientation(landscape: Boolean)
    fun onTopBarVisibility(visible: Boolean)
}